pkg/virt-controller/watch/migration.go

```diff
@@ -538,6 +538,7 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 				TargetNode:   pod.Spec.NodeName,
 				SourceNode:   vmi.Status.NodeName,
 				TargetPod:    pod.Name,
+				Mode:         virtv1.MigrationPreCopy,
```

Comment:
is it right that preCopy is set unconditionally here?

Reply:
This `MigrationState` is creating the initial migration state and since all migrations start in precopy mode I thought it was most accurate to have the mode in this object reflect that. The mode could instead not be set during the initial creation of the migration state object, then once the migration starts have it set?

Reply:
yeah, I think it would be more accurate to not set mode until we know what it is. For instance, If the UI ever displays the mode, and someone see's a value that don't expect there, it might be kind of confusing.